### PR TITLE
fix: fix padding on drawers on mobile

### PR DESCRIPTION
**Pull request by nensidosari**
## Changes
- added styles to choose between the max padding between env and a set value of 0.75rem, this way it works on all cases. (tested on real device and browser)

I tried to do it with tailwind like this `pb-[max(env(safe-area-inset-bottom,0.75rem), 0.75rem)]` but didn't work, that's why added module.css. If u have any better recommendation please advise 🙏 

### Preview domain
https://fix-drawer-mobile.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/drawers/Drawer.module.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.drawerPadding {
+  padding-bottom: max(env(safe-area-inset-bottom, 0.75rem), 0.75rem);
```
[the remark this answers is not in the document]
Comment (nensidosari) [reply packages/shared/src/components/drawers/Drawer.module.css:2]: Do u mean the whole `max(env(safe-area-inset-bottom, 0.75rem), 0.75rem)` ?

It's only used at the moment in footerNavbar `padding-bottom: env(safe-area-inset-bottom, 0);` and not 100% the same

[the remark this answers is not in the document]
Comment (rebelchris) [reply packages/shared/src/components/drawers/Drawer.module.css:2]: Think Lee has a good point, we can probably just move it to base.css and use from there?

**Review by rebelchris — APPROVED** [review]
Not breaking, but consideration for moving :)

**Comment by nensidosari** [comment]
@rebelchris @sshanzel moved both drawer and footer to base.css

Pr is approved so I can merge, but if u don't mind have a last look at it

**Review by rebelchris — COMMENTED** [review]
Looks great to me :)

**Pull request merged**